apis/database/v1beta1/rdsinstance_types.go

```diff
@@ -368,6 +399,10 @@ type RDSInstanceParameters struct {
 	// +optional
 	EngineVersion *string `json:"engineVersion,omitempty"`
 
+	// RestoreFrom specifies the details of the backup to restore when creating a new RDS instance. (If the RDS instance already exists, this property will be ignored.)
+	// +optional
+	RestoreFrom *RestoreBackupConfiguration `json:"restoreFrom"`
```

Comment:
```suggestion
	RestoreFrom *RestoreBackupConfiguration `json:"restoreFrom,omitempty"`
```

The optional fields should [have](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#optional-vs-required) `omitempty` tag.